### PR TITLE
CardConnect: add store and unstore

### PR DESCRIPTION
**Pull request by bpollack**
Unit: 21 tests, 89 assertions, 0 failures, 0 errors, 0 pendings, 0 omissions, 0 notifications
100% passed

Remote: 23 tests, 54 assertions, 0 failures, 0 errors, 0 pendings, 0 omissions, 0 notifications
100% passed

### CODE REVIEW

File: lib/active_merchant/billing/gateways/card_connect.rb [hunk 1]
```diff
@@ -2,7 +2,10 @@ module ActiveMerchant #:nodoc:
   module Billing #:nodoc:
     class CardConnectGateway < Gateway
       self.test_url = 'https://fts.cardconnect.com:6443/cardconnect/rest/'
+      # self.test_cardsecure_url = 'https://fts.cardconnect.com:6443/cardsecure/cs'
```
Comment (curiousepic) [inline lib/active_merchant/billing/gateways/card_connect.rb]: Do we want to keep this, or the url construction method that seems unused? Or is cardsecure integration still coming?

File: lib/active_merchant/billing/gateways/card_connect.rb [hunk 2]
```diff
@@ -271,7 +295,11 @@ def message_from(response)
       end
 
       def authorization_from(response)
-        response['retref']
+        if response['profileid']
+          "#{response['acctid']}||#{response['profileid']}"
```
Comment (curiousepic) [inline lib/active_merchant/billing/gateways/card_connect.rb]: Nitpick but this kind of makes it look like a concated split auth, when it isn't. Maybe pull the conditional outside of the interpolation string?

Comment (curiousepic) [reply lib/active_merchant/billing/gateways/card_connect.rb]: Oh, hah, it is concating. In that case, the opposite of what I said; most separators are single character.

File: lib/active_merchant/billing/gateways/card_connect.rb (hunk 1 above)
Comment (bpollack) [reply lib/active_merchant/billing/gateways/card_connect.rb]: Dammit, will axe. Thanks for the catch.

File: lib/active_merchant/billing/gateways/card_connect.rb (hunk 2 above)
Comment (bpollack) [reply lib/active_merchant/billing/gateways/card_connect.rb]: Just to be clear, just do `|` instead of `||`?

**Comment by bpollack** [comment]
Killed commented-out lines and switched separator from `||` to `|`. Unless I hear otherwise, I'll merge this when CI finishes.

File: lib/active_merchant/billing/gateways/card_connect.rb [hunk 3]
```diff
@@ -175,13 +190,19 @@ def add_invoice(post, options)
       end
 
       def add_payment(post, payment)
-        post[:name] = payment.name
-        if card_brand(payment) == 'check'
-          add_echeck(post, payment)
+        if payment.is_a?(String)
+          account_id, profile_id = payment.split('||')
```
Comment (curiousepic) [inline lib/active_merchant/billing/gateways/card_connect.rb]: This is still splitting on `||`

Comment (bpollack) [reply lib/active_merchant/billing/gateways/card_connect.rb]: Well that'll teach me not to run remote tests. Fixed (along with the corresponding test).

**Pull request closed without merging**